components/src/core/drawer/drawer-nav-item/drawer-nav-item.component.ts

```diff
@@ -0,0 +1,147 @@
+                matRipple
+                [matRippleDisabled]="!ripple"
+                [matRippleColor]="rippleColor"
+                style="display:flex;"
+            >
+                <div class="pxb-drawer-nav-item-icon-wrapper" icon>
+                    <ng-content select="[icon]"></ng-content>
+                </div>
+                <div title>{{ title }}</div>
+                <div subtitle>{{ subtitle }}</div>
+                <div rightContent *ngIf="hasChildren && expandIcon && !showNestedNavItems">
+                    <mat-icon
+                        class="pxb-drawer-nav-item-expand-icon"
+                        (click)="toggleNestedNavItems($event)"
+                        [@rotateExpandIconAnimation]
+                        [@.disabled]="!toggled || !useCustomIconAnimation"
+                        >{{ expandIcon }}</mat-icon
+                    >
+                </div>
+                <div rightContent *ngIf="hasChildren && collapseIcon && showNestedNavItems">
+                    <mat-icon
+                        class="pxb-drawer-nav-item-collapse-icon"
+                        (click)="toggleNestedNavItems($event)"
+                        [@rotateCollapseIconAnimation]
+                        [@.disabled]="!toggled || !useCustomIconAnimation"
+                        >{{ collapseIcon }}</mat-icon
+                    >
+                </div>
+                <div
+                    rightContent
+                    *ngIf="hasChildren && !expandIcon && !collapseIcon"
+                    (click)="toggleNestedNavItems($event)"
+                >
+                    <mat-icon
+                        class="pxb-drawer-nav-item-default-expand-icon"
+                        [ngClass]="showNestedNavItems ? 'inverted' : ''"
+                        >keyboard_arrow_down</mat-icon
+                    >
+                </div>
+            </pxb-info-list-item>
+        </div>
+        <!-- Nested Nav Items -->
+        <div class="pxb-drawer-nested-nav-item">
+            <div *ngIf="showNestedNavItems" [@showNestedItemsAnimation]>
+                <ng-content
+                    select="pxb-drawer-nav-item"
+                    [ngClass]="selected ? 'pxb-info-list-item-active' : ''"
+                ></ng-content>
+            </div>
+        </div>
+    `,
+})
+export class DrawerNavItemComponent {
+    @Input() statusColor: string;
+    @Input() title: string;
+    @Input() subtitle: string;
+    @Input() chevron = false;
+    @Input() dense = false;
+    @Input() divider = true;
+    @Input() selected: boolean;
+    @Input() itemID: string;
+    @Input() showNestedNavItems = false;
+    @Input() hasChildren = false;
+    @Input() ripple = true;
+    @Input() expandIcon: string;
+    @Input() collapseIcon: string;
+    @Input() useCustomIconAnimation = true;
+    @Input() hidePadding: boolean;
+    @Input() rippleColor: string;
```

Comment:
I think rippleColor is something that can be handled by a user via css; we shouldn't need a prop on the component for this.